tssc/step_implementers/static_code_analysis/sonarqube.py

```diff
@@ -0,0 +1,251 @@
+"""Step Implementer for the static-code-analysis step for SonarQube.
+
+Step Configuration
+------------------
+
+Step configuration expected as input to this step.
+Could come from either configuration file or
+from runtime configuration.
+(Reference:  https://docs.sonarqube.org/latest/analysis/analysis-parameters)
+
+| Configuration Key | Required? | Default | Description
+|-------------------|-----------|---------|-----------
+| `properties'      | True      | None    | Properties file (./sonar-project.properties)
```

Comment:
Maybe default this to `./sonar-project.properties`?

Reply:
fysa - 

- the reference project has sonar-project.properties in the cicd folder with cicd/sonar-project.properties in the tssc-config.yml
- default is set to to sonar-project.properties (current folder); if the user currently has sonar-qube properties, most likely it is in the root folder